src/components/RadioButton/RadioButton.jsx

```diff
@@ -0,0 +1,125 @@
+import _ from 'lodash';
+import classNames from 'classnames';
+import React from 'react';
+
+import { bindClassNames } from '../../util/style-helpers';
+
+const generateBoundClassNames = bindClassNames('RadioButton');
```

Comment:
`Verbose and explicit` > `concise and unclear` ;)

Actually `boundClassNames` doesn't seem too ambiguous to me, but it breaks the general rule of thumb that a function/method should be named as an action verb.

But if the team feels strongly enough that `boundClassNames` is fine I'll begrudgingly concede. ;)